src/Process/Runner/ParallelProcessRunner.php

```diff
@@ -175,6 +175,8 @@ private function tryToFreeNotRunningProcess(): ?MutantProcessContainer
             }
 
             if (!$process->isRunning()) {
+                $mutantProcess->markAsFinished();
```

Comment:
btw, it looks like at this point of time process could already be finished and sitting "waiting" for us unless this function is executed.

So we potentially can see bigger numbers that process took in reality. However, I don't see a better approach for now, just sating